topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -988,55 +1415,207 @@ goseq needs 2 files as inputs:
+>
+> In this tutorial, we covered GO enrichment analysis with **goseq**. To learn other gene set enrichment analysis, please have a look at the ["RNA-seq genes to pathways"]({{ site.baseurl }}{% link topics/transcriptomics/tutorials/rna-seq-genes-to-pathways/tutorial.md %}) tutorial.
+{: .comment}
+
+### KEGG pathways
+
+**goseq** can also be used to identify interesting KEGG pathways. The KEGG pathway database is a collection of pathway maps representing the current knowledge on the molecular interaction, reaction and relation networks. A map can integrate many entities including genes, proteins, RNAs, chemical compounds, glycans, and chemical reactions, as well as disease genes and drug targets. 
+
+For example, the pathway `dme00010` represents the Glycolysis process (conversion of glucose into pyruvate with generation of small amounts of ATP and NADH) for Drosophila melanogaster:
+
+![dme00010 KEGG pathway](../../images/ref-based/dme00010_empty.png)
+
+> ### {% icon hands_on %} Hands-on: Perform GO analysis
+>
+> 1. **goseq** {% icon tool %} with
+>    - *"Differentially expressed genes file"*: `Gene IDs and differentially expression`
+>    - *"Gene lengths file"*: `Gene IDs and length`
+>    - *"Gene categories"*: `Get categories`
+>       - *"Select a genome to use"*: `Fruit fly (dm6)`
+>       - *"Select Gene ID format"*: `Ensembl Gene ID`
+>       - *"Select one or more categories"*: `KEGG`
+>    - In *"Output Options"*
+>      - *"Output Top GO terms plot?"*: `No`
+>      - *"Extract the DE genes for the categories (GO/KEGG terms)?"*: `Yes`
+>
+{: .hands_on}
 
-To identify categories significantly enriched/unenriched below some p-value cutoff, it is necessary to use the adjusted *p*-value.
+As for the GO terms, 2 files are generated:
+
+1. A big table with the KEGG terms and some statistics
+
+    > ### {% icon question %} Questions
+    >
+    > 1. How many KEGG pathways terms have been identified?
+    > 2. How many KEGG pathways terms are over-represented with an adjusted P value < 0.05?
+    > 3. What are the over-represented KEGG pathways terms?
+    > 4. How many KEGG pathways terms are under-represented with an adjusted P value < 0.05?
+    >
+    > > ### {% icon solution %} Solution
+    > >
+    > > 1. The file has 127 lines including an header, so 126 KEGG pathways have been identified.
+    > > 2. 2 KEGG pathways (1.57%) are over-represented, using **Filter** {% icon tool %} on c6 (adjusted p-value for over-represented KEGG pathways)
+    > > 3. The first 3 KEGG pathways are:
+    > >    - `01100`: Metabolic pathways, all metabolic pathways
+    > >    - `00010`: Glycolysis / Gluconeogenesis
+    > > 4. 1 KEGG pathways (0.79%) is under-represented, using **Filter** {% icon tool %} on c7 (adjusted p-value for under-represented KEGG pathways): `03040`, Spliceosome
+    > {: .solution}
+    {: .question}
+
+2. A table with the differentially expressed genes (from the list we provided) associated to the KEGG pathways
```

Comment:
output is called "DE genes for categories (GO/KEGG terms)"

Reply:
Added